src/checker/scripts/junit.policy

```diff
@@ -18,5 +24,20 @@ grant {
 	//to enable JUNIT and JUNIT-PARAMS Tests for nested unnamed classes
 	permission java.lang.RuntimePermission "createSecurityManager";
 	permission java.lang.RuntimePermission "setSecurityManager";
-	permission java.io.FilePermission "/home/praktomat/inst/2021t/work-data/SolutionSandbox/*", "read"; //TODO: update for every instance: how we can do this more generic?
+};
+
+// Settings for own java-based TestFrameworks started via JUnitChecker or JavaChecker
+
+grant codeBase "file:db-checker-praktomat-1.0-SNAPSHOT.jar" {
```

Comment:
Looks like this doesn't belong here and should only be a local change of yours, right? This file is not present anywhere in the Praktomat repository. Same applies for a lot of the changes in `java.policy`.

Reply:
> I wonder if we should actually introduce these workarounds. The security manager is already deprecated and will be removed at some point in the future. We need to remove it anyway, why not now? When running with Docker, the environment is encapsulated anyway. 

We introduce the workaround, because the deprecation warnings of security manager confused some people, who thought they had something to do with their solutions. 

Encapsulating via Docker can not prevent students to walk the Solution-Sandbox and print out testcases. Or did you solve that "problem" it without JVM Security manager?

Reply:
Hannes Braun worte a comment about changes in `java.policy `and `junit.policy`:
> Looks like this doesn't belong here and should only be a local change of yours, right? This file is not present anywhere in the Praktomat repository. Same applies for a lot of the changes in java.policy.

Well, You are mostly right:  
The configuration for `db-checker-praktomat-1.0-SNAPSHOT.jar` in `junit.policy` 
and `TestRunner.jar`, `TestSuite.jar` in `java.policy` should be in our local changes. 
But if commenting them out, than they could be an example how to add own additional entries.
Indeed that is obvious only usefull, for the time in which JVM security Manager can be used.

Reply:
I changed that configuration lines in `junit.policy` and `java.policy` to become comment blocks. Feel free to edit my proposed changes to your needs, e.g delete them if you wish :-)